.github/workflows/continuous-integration.yaml

```diff
@@ -75,15 +76,33 @@ jobs:
             elasticsearch: '8.10.4' # there are no bc in minor version https://www.elastic.co/guide/en/elasticsearch/reference/current/migrating-8.10.html
             experimental: false
           - php: '8.1'
-            elasticsearch: '8.11.4' # there are some bc in minor version https://www.elastic.co/guide/en/elasticsearch/reference/current/migrating-8.10.html
+            elasticsearch: '8.11.4' # there are some bc in minor version https://www.elastic.co/guide/en/elasticsearch/reference/current/migrating-8.11.html
             experimental: false
           - php: '8.1'
-            elasticsearch: '8.12.0' # there are some bc in minor version https://www.elastic.co/guide/en/elasticsearch/reference/current/migrating-8.12.html
+            elasticsearch: '8.12.2' # there are some bc in minor version https://www.elastic.co/guide/en/elasticsearch/reference/current/migrating-8.12.html
+            experimental: false
+          - php: '8.1'
+            elasticsearch: '8.13.4' # there are some bc in minor version https://www.elastic.co/guide/en/elasticsearch/reference/current/migrating-8.13.html
+            experimental: false
+          - php: '8.1'
+            elasticsearch: '8.14.3' # there are some bc in minor version https://www.elastic.co/guide/en/elasticsearch/reference/current/migrating-8.14.html
+            experimental: false
+          - php: '8.1'
+            elasticsearch: '8.15.5' # there are some bc in minor version https://www.elastic.co/guide/en/elasticsearch/reference/current/migrating-8.15.html
+            experimental: false
+          - php: '8.1'
+            elasticsearch: '8.16.6' # there are some bc in minor version https://www.elastic.co/guide/en/elasticsearch/reference/current/migrating-8.16.html
+            experimental: false
+          - php: '8.1'
```

Comment:
Should we really test against all the older minor elasticsearch versions? I would focus either only on the last minor or the two last minors.

Reply:
Fine for me. Changed.

I left one more version: ES 7.17.28.